### DISCUSSION TITLE
Generate RC files for windows DLLs

### DISCUSSION THREAD

**Discussion by jmklix** [aws/aws-sdk-cpp#3940] [General]
### Describe the feature

original discussion: https://github.com/aws/aws-sdk-cpp/discussions/2887#discussioncomment-8876613

generate RC files for windows DLLs to include the product information
 - Company name
 - Product Version

### Use Case

it would help determine which version is deployed and other information about this sdk

### Acknowledgements

- [ ] I may be able to implement this feature request
- [ ] This feature might incur a breaking change

**Comment by kai-ion** [comment]
Converting this to a Discussion. It's an improvements we plan to pursue over time rather than one actionable change with a near-term milestone, so a Discussion better fits collecting the specifics and design input.
